Examples/Tutorials/Tutorial-LiveTraffic/main.cpp

```diff
@@ -74,7 +72,7 @@ struct PacketStats
 static void onPacketArrives(pcpp::RawPacket* packet, pcpp::PcapLiveDevice* dev, void* cookie)
 {
 	// extract the stats object form the cookie
-	PacketStats* stats = (PacketStats*)cookie;
+	auto* stats = static_cast<PacketStats*>(cookie);
```

Comment:
`auto` here make sense.

Reply:
IMO, the sensible thing here is to use static casting instead of c-style casting.